src/validation.h

```diff
@@ -289,6 +289,24 @@ PackageMempoolAcceptResult ProcessNewPackage(CChainState& active_chainstate, CTx
  */
 bool CheckFinalTxAtTip(const CBlockIndex* active_chain_tip, const CTransaction& tx) EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 
+/**
+ * Calculate LockPoints required to check if transaction will be BIP68 final in the next block to be created on top of tip.
+ * @param[in]   tip             Chain tip to check tx sequence locks against. For example,
+ *                              the tip of the current active chain.
+ * @param[in]   coins_view      Any CCoinsView that provides access to the relevant coins for
+ *                              checking sequence locks. For example, it can be a CCoinsViewCache
+ *                              that isn't connected to anything but contains all the relevant
+ *                              coins, or a CCoinsViewMemPool that is connected to the
+ *                              mempool and chainstate UTXO set. In the latter case, the caller is
+ *                              responsible for holding the appropriate locks to ensure that
+ *                              calls to GetCoin() return correct coins.
+ *
+ * @returns The resulting height and time calculated and the hash of the block needed for calculation.
```

Comment:
in fc31aa12b9599f3a23c22fd65b3eaa73dbb05d81:

Should document that this returns std::nullopt if there is an error, i.e., if inputs cannot be found in the `coins_view`

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23897#issuecomment-1191310762).